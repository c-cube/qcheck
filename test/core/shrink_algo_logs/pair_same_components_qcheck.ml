open QCheck

let pair_same =
  Test.make ~name:"pairs have same components"
    (pair int int) (Log.shrinks Print.(pair int int) @@ fun (i,j) -> i=j)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand pair_same with Test.Test_fail _ -> ()
