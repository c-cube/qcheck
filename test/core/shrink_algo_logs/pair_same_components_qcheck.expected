fails (-4571327332697646483, -3285013039971199785)
fails (-2285663666348823242, -3285013039971199785)
fails (-1142831833174411621, -3285013039971199785)
fails (-571415916587205811, -3285013039971199785)
fails (-285707958293602906, -3285013039971199785)
fails (-142853979146801453, -3285013039971199785)
fails (-71426989573400727, -3285013039971199785)
fails (-35713494786700364, -3285013039971199785)
fails (-17856747393350182, -3285013039971199785)
fails (-8928373696675091, -3285013039971199785)
fails (-4464186848337546, -3285013039971199785)
fails (-2232093424168773, -3285013039971199785)
fails (-1116046712084387, -3285013039971199785)
fails (-558023356042194, -3285013039971199785)
fails (-279011678021097, -3285013039971199785)
fails (-139505839010549, -3285013039971199785)
fails (-69752919505275, -3285013039971199785)
fails (-34876459752638, -3285013039971199785)
fails (-17438229876319, -3285013039971199785)
fails (-8719114938160, -3285013039971199785)
fails (-4359557469080, -3285013039971199785)
fails (-2179778734540, -3285013039971199785)
fails (-1089889367270, -3285013039971199785)
fails (-544944683635, -3285013039971199785)
fails (-272472341818, -3285013039971199785)
fails (-136236170909, -3285013039971199785)
fails (-68118085455, -3285013039971199785)
fails (-34059042728, -3285013039971199785)
fails (-17029521364, -3285013039971199785)
fails (-8514760682, -3285013039971199785)
fails (-4257380341, -3285013039971199785)
fails (-2128690171, -3285013039971199785)
fails (-1064345086, -3285013039971199785)
fails (-532172543, -3285013039971199785)
fails (-266086272, -3285013039971199785)
fails (-133043136, -3285013039971199785)
fails (-66521568, -3285013039971199785)
fails (-33260784, -3285013039971199785)
fails (-16630392, -3285013039971199785)
fails (-8315196, -3285013039971199785)
fails (-4157598, -3285013039971199785)
fails (-2078799, -3285013039971199785)
fails (-1039400, -3285013039971199785)
fails (-519700, -3285013039971199785)
fails (-259850, -3285013039971199785)
fails (-129925, -3285013039971199785)
fails (-64963, -3285013039971199785)
fails (-32482, -3285013039971199785)
fails (-16241, -3285013039971199785)
fails (-8121, -3285013039971199785)
fails (-4061, -3285013039971199785)
fails (-2031, -3285013039971199785)
fails (-1016, -3285013039971199785)
fails (-508, -3285013039971199785)
fails (-254, -3285013039971199785)
fails (-127, -3285013039971199785)
fails (-64, -3285013039971199785)
fails (-32, -3285013039971199785)
fails (-16, -3285013039971199785)
fails (-8, -3285013039971199785)
fails (-4, -3285013039971199785)
fails (-2, -3285013039971199785)
fails (-1, -3285013039971199785)
fails (0, -3285013039971199785)
fails (0, -1642506519985599893)
fails (0, -821253259992799947)
fails (0, -410626629996399974)
fails (0, -205313314998199987)
fails (0, -102656657499099994)
fails (0, -51328328749549997)
fails (0, -25664164374774999)
fails (0, -12832082187387500)
fails (0, -6416041093693750)
fails (0, -3208020546846875)
fails (0, -1604010273423438)
fails (0, -802005136711719)
fails (0, -401002568355860)
fails (0, -200501284177930)
fails (0, -100250642088965)
fails (0, -50125321044483)
fails (0, -25062660522242)
fails (0, -12531330261121)
fails (0, -6265665130561)
fails (0, -3132832565281)
fails (0, -1566416282641)
fails (0, -783208141321)
fails (0, -391604070661)
fails (0, -195802035331)
fails (0, -97901017666)
fails (0, -48950508833)
fails (0, -24475254417)
fails (0, -12237627209)
fails (0, -6118813605)
fails (0, -3059406803)
fails (0, -1529703402)
fails (0, -764851701)
fails (0, -382425851)
fails (0, -191212926)
fails (0, -95606463)
fails (0, -47803232)
fails (0, -23901616)
fails (0, -11950808)
fails (0, -5975404)
fails (0, -2987702)
fails (0, -1493851)
fails (0, -746926)
fails (0, -373463)
fails (0, -186732)
fails (0, -93366)
fails (0, -46683)
fails (0, -23342)
fails (0, -11671)
fails (0, -5836)
fails (0, -2918)
fails (0, -1459)
fails (0, -730)
fails (0, -365)
fails (0, -183)
fails (0, -92)
fails (0, -46)
fails (0, -23)
fails (0, -12)
fails (0, -6)
fails (0, -3)
fails (0, -2)
fails (0, -1)
holds (0, 0)
